### PR TITLE
.net sdk

### PR DESCRIPTION
**Pull request by barnwal-paras**

### CODE REVIEW

File: RestComm/NUnitTestClass.cs [hunk 1]
```diff
@@ -0,0 +1,16 @@
+﻿using NUnit.Framework;
+using System;
+
+namespace RestComm
+{
+	[TestFixture ()]
```
Comment (gvagenas) [inline RestComm/NUnitTestClass.cs:6]: The `TestFixture` gives me the following error when open the project with VisualStudio on Mac
```
/Users/gvagenas/data/devWorkspace/telWorkspace/restcomm-sdk-dotnet/RestComm/NUnitTestClass.cs(3,3): Error CS0619: 'TestFixtureAttribute' is obsolete: 'The NUnit framework shipped with Mono is deprecated and will be removed in a future release. It was based on NUnit 2.4 which is long outdated. Please move to the NUnit NuGet package or some other form of acquiring NUnit.' (CS0619) (RestComm)
```

File: RestComm/NUnitTestClass.cs [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+	public class NUnitTestClass
```
Comment (gvagenas) [inline RestComm/NUnitTestClass.cs:7]: You should provide unit tests for all of the SDK features

**Review by gvagenas — CHANGES_REQUESTED** [review]
Hi @barnwal-paras , great work so far! 

Since I am not a dotnet expert, I will provide my comments for discussion and you can suggest what should better fit the dotnet project nature and standards followed here.

Following my comments:
- Separate unit tests from actual classes
- Provide documentation for the SDK. For the Java project we use Asciidoc and I believe you can use the same here. The benefits of Asciidoc is that it can be converted to pdf, html etc. We use HTML to publish documentation online.
- Please include a simple tutorial, how a developer should use the SDK, how to include the dependency in the app, how to write a simple app etc.

**Comment by gvagenas** [comment]
Also, important is to include LICENSE header on all files, and the LICENSE file at the root of the project. 
```
/*
 * TeleStax, Open Source Cloud Communications
 * Copyright 2011-2016, Telestax Inc and individual contributors
 * by the @authors tag.
 *
 * This is free software; you can redistribute it and/or modify it
 * under the terms of the GNU Lesser General Public License as
 * published by the Free Software Foundation; either version 2.1 of
 * the License, or (at your option) any later version.
 *
 * This software is distributed in the hope that it will be useful,
 * but WITHOUT ANY WARRANTY; without even the implied warranty of
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
 * Lesser General Public License for more details.
 *
 * You should have received a copy of the GNU Lesser General Public
 * License along with this software; if not, write to the Free
 * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
 * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
 */
```
* Eclipse: 
Preferences -> Java -> Code Style -> Code Templates -> Comments -> Files -> Pattern
* Intellij Idea
Settings -> Editor -> Copyright and Copyright profiles

**Comment by barnwal-paras** [comment]
@gvagenas ,I am closing this pull request . I will add the changes suggested by you in my other PR.
Thanks .

File: RestComm/NUnitTestClass.cs (hunk 1 above)
Comment (barnwal-paras) [reply RestComm/NUnitTestClass.cs:6]: I was using Monodevelop with Nunit version 2.6.4(this is latest version of nunit v2) . Can you try once in monodevelop and tell me if you are still getting same error .This problem might be specific to visual studio only .

**Pull request closed without merging**